NuGet.config

```diff
@@ -10,8 +10,8 @@
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
     <add key="vs-buildservices" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
     <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
-    <add key="myget-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
-    <add key="myget-roslyn" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
+    <add key="mygetlegacy-core" value="https://dotnetmygetlegacy.blob.core.windows.net/dotnet-core/index.json" />
```

Comment:
What packages are we relying on that are only on this feed? We should see what packages are needed from the URL and find a different feed for them.

Reply:
So I removed it, cleared out my packages, and rebuilt locally. No errors. I think that line 12 was added in to replace it, and then it was never actually removed. I have now removed it completely. Since it built locally fine I am assuming it will as well in the CI, but at least we will be able to test all configurations this way.

Reply:
All builds have passed the build step. I am going to assume that removing it was fine and mark this as resolved.